src/vs/workbench/parts/files/electron-browser/fileActions.ts

```diff
@@ -1311,7 +1309,7 @@ export class ShowActiveFileInExplorer extends Action {
 	public run(): TPromise<any> {
 		const resource = toResource(this.editorService.getActiveEditorInput(), { supportSideBySide: true });
 		if (resource) {
-			this.commandService.executeCommand(REVEAL_IN_EXPLORER_COMMAND_ID, { resource });
+			this.commandService.executeCommand(REVEAL_IN_EXPLORER_COMMAND_ID, resource, this.focusSelected);
```

Comment:
Note the change in arguments here. Passing `resource` as an property on an object isn't actually supported by the reveal in explorer command, seems to be a bug introduced by https://github.com/Microsoft/vscode/pull/40168.